assets/src/utils/kubernetes.ts

```diff
@@ -9,6 +9,13 @@ const MULTIPLES = {
 
 const nanToUndef = val => (typeof val !== 'number' || Number.isNaN(val) ? undefined : val)
 
+export function isEqual({ metadata: firstMeta }, { metadata: secondMeta }) {
+  if (firstMeta.namespace !== secondMeta.namespace) return false
+  if (firstMeta.name !== secondMeta.name) return false
+
+  return true
```

Comment:
could be shortened to `return firstMeta.name === secondMeta.name`

Reply:
yea i was worried about line length but i can make it shorter